codecov.yml

```diff
@@ -1,6 +1,6 @@
 codecov:
   notify:
-    after_n_builds: 10
+    after_n_builds: 20
```

Comment:
why ?

Reply:
Because I add the same item towards arm64 in .travis.yml config file, which has the same test for the default arch `amd64`. The reason to do this is trying the best to test the same things(test cases) with X86 for ARM. 
And for the check https://github.com/pgjdbc/pgjdbc/pull/1900/files#diff-354f30a63fb0907d4ad57269548329e3L24 , this cmd check whether the count of `COVERAGE=Y` in .travis.yml is equal to the setted `10` which set in codecov.yml. So replies the `matrix` part of .travis.yml config file(I add the same test item with amd64), the count should be doubled, it's 20. 
For now, this is my tried. Not sure changing the said value from `10` to `20` could raise other issues. So could you please give some advices? Thank you

Reply:
We use code coverage to track missing unit tests, etc. We do not have different code paths for arm64 vs x86, so I guess new Travis jobs can be added without `coverage=y`.

Reply:
@vlsi Thanks for advices. I will update it.